### ISSUE TITLE
A11y_Azure Tools for VS Code [Benchmark]_View Properties_Screen Reader: VoiceOver is not reading about Lines Information and Collapsed lines information in the code editor section

### ISSUE THREAD

**Issue by Ashwinipola26** [microsoft/vscode-azurefunctions#2693]
### Actual Result:
VoiceOver is not reading about Lines Information in the code editor section and also not reading about Collapsed lines information

### Expected Result:
VoiceOver should be reading about Lines Information and also about the Collapsed lines information in the code lines section

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yWCAG2.1; #A11yTCS; #AILimited; #Benchmark-Azure Tools for VS Code-Win-Feb2021; #DesktopApp;#WCAG1.3.1;#MAC; #ScreenReader;#Benchmark;#VoiceOver

### OS Details:
macOS Catalina Version 10.15.7

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z(6 days ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin X64 19.6.0
Azure Function Version: 1.2.1

### Repro Steps:

1. Start VoiceOver
2. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab till Create a Project and create one project
8. Tab till Created project and press Ctrl+left click and press enter
9. Tab till View Properties
10. Tab till Code Editor section check whether VoiceOver is reading about lines numbers or not.

### User Impact:
Screen Reader users will not know about the lines numbers and Collapsed line information

### Attachment for Reference:
